modyn/storage/src/CMakeLists.txt

```diff
@@ -135,4 +135,4 @@ target_compile_definitions(modyn-storage-library PUBLIC ${MODYN_COMPILE_DEFINITI
 set(INCLUDE_EXPR "$<TARGET_PROPERTY:modyn-storage-library,INCLUDE_DIRECTORIES>")
 set(INCLUDE_FILTER "$<FILTER:${INCLUDE_EXPR},INCLUDE,.*/.*>")
 set(INCLUDE_JOINED "$<JOIN:${INCLUDE_FILTER},,>")
-target_compile_definitions(modyn-storage-library PRIVATE "INCLUDE_DIRECTORIES=\"${INCLUDE_JOINED}\"")
```

Comment:
let's not touch this file with this commit :)

Reply:
Ok, I reverted these changes. Going forward, would you prefer file endings with or without newlines?